composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2",
+        "symfony/symfony": "~2|~3.0",
```

Comment:
Since https://github.com/michelsalib/BCCCronManagerBundle/pull/65, symfony minimum version is 2.2

`"symfony/symfony": "~2.2|~3.0",`